src/openassets.es6

```diff
@@ -29,14 +29,16 @@ module.exports = function(robot) {
   // Address Book
   //
 
+  let robotKeyword = process.env.OA_BOT_KEYWORD || 'kredits';
```

Comment:
how would I make it configurable per room with having the config not in the brain?